engineering-team/index.html

```diff
@@ -0,0 +1,270 @@
+<script src="../assets/js/global-custom-js.js"></script>
+<!-- Script for static navbar and footer -->
+<script src="../assets/js/navigation.js"></script>
+<script src="../assets/js/functions.js"></script>
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
+<script type="text/html" id="template-posts">
+    <div class="row">
+        {{#data}}
+        <div class="col-md-4 mb-5">
+            <div class="card px-0">
+                <a href="{{url}}" target="_blank" class="card shadow project-card">
+                    <div class="card-body posts-card-body">
+                        <img src="{{image}}" class="img-fluid  img-full" alt="Rounded image">
+                        <p class="card-text text-darker text-left ml-3">
+                            {{title}}
+                        </p>
+                    </div>
+                    <div class="card-footer">
+                        <div class="author">
+                            <img src="{{author_image}}" alt="..." class="avatar img-raised">
+                            <span>{{author_name}}</span>
+                        </div>
+                    </div>
+                </a>
+            </div>
+        </div>
+        {{/data}}
+    </div>
+</script>
+<script id="template-dev" type="text/html">
+    {{#card}}
+    <div class="project-card shadow my-4 ">
+        <div class="container">
+            <div class="row  py-5">
+                <div class="col-md-3 align-items-center justify-content-center text-center my-auto">
+                    <img src="{{image}}}"
+                         style="width: 100px; height: 100px; border-radius: 50%; border: 5px solid #EAECEE;">
+                    <a href="{{linkedin}}}" target="_blank"><p class="pt-2 font pr-2">{{dev_name}}</p></a>
+                </div>
+                <div class="col-md-9 font-size">
+                    <h4>"{{story}}"
+                    </h4>
+                </div>
+            </div>
+            <div class="pb-4 mx-4">
+                <h5>Acheivements</h5>
+                <ul>
+                    {{#acheivements}}
+                    <li>{{{.}}}</li>
+                    {{/acheivements}}
+                </ul>
+            </div>
+        </div>
+    </div>
+    {{/card}}
+</script>
+<script>
+    async function loadStories() {
+        const response = await fetch("https://script.google.com/macros/s/AKfycbwv81hbfTsHAGcNQnmrjcV9YPVkipHRbjXct_6DlPWrovCVqy90j8DlRNb2D_MTQAiOuQ/exec");
+        const data = await response.json();
+        const template = document.getElementById("template-dev").innerHTML;
+        const content = Mustache.render(template, {card: data});
+        document.getElementById('stories').innerHTML = content;
+    }
+
+    loadStories();
+</script>
+<script>
+    $(function () {
+        loadNavAndFooter('../assets/content/static');
+        // loadBlogPosts();
```

Comment:
Why this is commented?

Reply:
It was already commented

Reply:
Remove it then

Reply:
Remove the empty `engineering-team/styles.css` file as well

Reply:
Okay